tsfresh/utilities/distribution.py

```diff
@@ -86,13 +89,18 @@ def partition(data, chunk_size):
         :rtype: generator
```

Comment:
I can't comment the docstring, but it says that 

> If the chunk_size is not a divider of the data length, the last slice will be shorter than chunk_size.

This is not correct anymore, is it? The last chunk will be larger than the chunk_size, right?

Reply:
No. Why so you think so? Also, it would be an error to return more items than the input data has.